CMakeLists.txt

```
@@ -18,7 +18,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
 endif()
 
 add_library(peglib INTERFACE)
-target_include_directories(peglib INTERFACE ${CMAKE_SOURCE_DIR})
+target_include_directories(peglib INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

```suggestion
target_include_directories(peglib INTERFACE
  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
  $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>
)
```

We should separate include directories for build/install.

---

Does it even matter if the lib isn't installed? Only the header itself is installed, there's no export for the lib.

---

Oh I didn't check the whole `CMakeLists.txt`. This change has no effect but I suggested it routinely.

---

@jimmy-park so can we merge the original change by @trcwm as it is?

---

@yhirose Yes, please ignore this suggestion.